boards/infineon/cyw920829m2evk_02/doc/index.rst

```diff
@@ -0,0 +1,167 @@
+
+
+Kit Features:
+=============
+
+- AIROC™ CYW20829 Bluetooth® LE MCU in 56 pin QFN package
+- Arduino compatible headers for hardware expansion
+- On-board sensors - 6-axis IMU, thermistor, Infineon analog microphone, and Infineon digital microphone
+- User switches, RGB LED and user LEDs
+- USB connector for power, programming and USB-UART bridge
+
+Kit Contents:
+=============
+
+- CYW20829 evaluation board (CYW9BTM2BASE3+CYW920829M2IPA2)
+- USB Type-A to Micro-B cable
+- Six jumper wires (five inches each)
+- Quick start guide
+
+
+Supported Features
+==================
+
+The board configuration supports the following hardware features:
+
++-----------+------------+-----------------------+
+| Interface | Controller | Driver/Component      |
++===========+============+=======================+
+| NVIC      | on-chip    | nested vectored       |
+|           |            | interrupt controller  |
++-----------+------------+-----------------------+
+| SYSTICK   | on-chip    | system clock          |
++-----------+------------+-----------------------+
+| GPIO      | on-chip    | GPIO                  |
++-----------+------------+-----------------------+
+| UART      | on-chip    | serial port-polling;  |
+|           |            | serial port-interrupt |
++-----------+------------+-----------------------+
+
+
+The default configuration can be found in the Kconfig
+
+:zephyr_file:`boards/infineon/cyw920829m2evk_02/cyw920829m2evk_02_defconfig`
+
+
+System Clock
+============
+
+The AIROC™ CYW20829 Bluetooth®  MCU SoC is configured to use the internal IMO+FLL as a source for
+the system clock. Other sources for the system clock are provided in the SOC, depending on your
+system requirements.
+
+
+Fetch Binary Blobs
+******************
+
+cyw920829m2evk_02 board optionally uses binary blobs for Bluetooth features.
+
+To fetch Binary Blobs:
+
+.. code-block:: console
+
+   west blobs fetch hal_infineon
+
+
+Build blinking led sample
+*************************
+
+Here is an example for the :zephyr:code-sample:`blinky` application.
+
+.. code-block:: console
+
+   cd zephyr
+   west build -p auto -b cyw920829m2evk_02 samples/basic/blink
+
+SREC_CAT Installation
+*********************
+
+The ``srec_cat`` utility is used to generate the application header (post build). Install the utility using the below steps depending on the OS
+
+This can be installed on Linux via:
+
+   .. code-block:: bash
+
+      $ sudo apt-get install srecord
+
+And on OS X via:
+
+   .. code-block:: bash
+
+      $ brew install srecord
+
+For Windows-based systems, please make sure you have a copy of the utility
+available on your system path. See, for example:
+`SRecord for Windows <https://sourceforge.net/projects/srecord/files/srecord-win32>`_
+
+OpenOCD Installation
+====================
+
+To get the OpenOCD package, it is required that you
+
+1. Download the software ModusToolbox 3.1. https://softwaretools.infineon.com/tools/com.ifx.tb.tool.modustoolbox
```

Comment:
Might as well change this to ModusToolbox 3.2, since that came out a few weeks ago. Another option is to say ModusToolbox 3.1 or greater, so we don't have to change when new releases come out.